src/lib/db/group-store.ts

```diff
@@ -33,6 +33,8 @@ const GROUP_COLUMNS = [
     'scim_id',
 ];
 
+export const SSO_SYNC_USER = 'SSO';
```

Comment:
I'm making this very explicit cause it gave me a headache to realize this username **has to be used** in the sync groups workflow instead of the audit user username (but only to call this store)